### PR TITLE
Update orders table

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://projects.engineering.redhat.com/browse/SSP-1695

### Changes
- switch the orders table layout once again
- added sorting to the table
  - currently, we can only sort by id, owner, state and created
- the status label is no longer a link

new-order-table

### CODE REVIEW

**Comment by Hyperkid123** [comment]
cc @fionayhlin

File: src/smart-components/order/order-item.js [hunk 1]
```diff
@@ -1,155 +1,121 @@
...
+        <Text className="pf-u-mb-0" component={TextVariants.small}>
+          <DateFormat date={item.created_at} variant="relative" />
+        </Text>
+      )
+    },
+    {
+      title: (
+        <Text className="pf-u-mb-0" component={TextVariants.small}>
+          {formatMessage(ordersMessages.lastUpdated)}
+          &nbsp;
```
Comment (rvsia) [inline src/smart-components/order/order-item.js:96]: Can we put classname with a modifier to the next DateFormat?

File: src/smart-components/order/orders-list.js [hunk 2]
```diff
@@ -233,66 +309,47 @@ const OrdersList = () => {
...
+                    {meta.noData
+                      ? formatMessage(ordersMessages.noOrdersTitle)
+                      : formatMessage(filteringMessages.noResults)}
+                  </Title>
+                  <EmptyStateBody>
+                    {meta.noData
+                      ? formatMessage(ordersMessages.noOrdersDescription)
+                      : formatMessage(filteringMessages.noResultsDescription)}
+                  </EmptyStateBody>
+
```
Comment (rvsia) [inline src/smart-components/order/orders-list.js:340]: empty line

File: src/smart-components/order/orders-list.js [hunk 3]
```diff
@@ -233,66 +309,47 @@ const OrdersList = () => {
...
+                  <EmptyStateSecondaryActions>
+                    {!meta.noData && (
```
Comment (rvsia) [inline src/smart-components/order/orders-list.js:342]: what about to level-up this condition so `EmptyStateSecondaryActions` won't be empty?

File: src/smart-components/order/order-item.js (hunk 1 above)
Comment (Hyperkid123) [reply src/smart-components/order/order-item.js:96]: No need the last updated string should be removed!

**Comment by Hyperkid123** [comment]
@Hyperkid123 all should be fine now.

**Comment by Hyperkid123** [comment]
@rvsia I see your point. It's only doable by changing the columns definition. I have purposely put that inside a ref so it won't create the columns on each render. If only there was a feature to turn it off on the table level :no_mouth:. I will change it.

**Comment by Hyperkid123** [comment]
I have forgot to add the sort fragment to the mocked URLs

**Pull request merged**